README.markdown

```diff
@@ -220,16 +220,37 @@ connect
 
 `syntax: ok, err = red:connect("unix:/path/to/unix.sock", options_table?)`
 
+
 Attempts to connect to the remote host and port that the redis server is listening to or a local unix domain socket file listened by the redis server.
 
 Before actually resolving the host name and connecting to the remote backend, this method will always look up the connection pool for matched idle connections created by previous calls of this method.
 
-An optional Lua table can be specified as the last argument to this method to specify various connect options:
+The optional `options_table` argument is a Lua table holding the following keys:
+
+* `ssl`
+
+    If set to true, then uses SSL to connect to redis (default to false)
+
+* `ssl_verify`
+
+    If set to true, then verifies the validity of the server SSL certificate (default to false). Note that you need to configure the lua_ssl_trusted_certificate to specify the CA (or server) certificate used by your redis server. You may also need to configure lua_ssl_verify_depth accordingly.
+
+* `server_name`
+
+    server name argument is used to specify the server name for the new TLS extension Server Name Indication (SNI). Defaults to `host`
 
 * `pool`
 
     Specifies a custom name for the connection pool being used. If omitted, then the connection pool name will be generated from the string template `<host>:<port>` or `<unix-socket-path>`.
 
+* `pool_size`
```

Comment:
I noticed your README lists `pool_size` as an option, but in the code, the name of the option is `poolsize`.

Reply:
thanks, updated with correct option.